lib/index.js

```diff
@@ -58,8 +58,10 @@ api.db = null;
 // shared collections cache
 api.collections = {};
 
+const {writeConcern} = bedrock.config.mongodb.writeOptions;
```

Comment:
This isn't safe to use and should be removed. It can change during bedrock startup.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-mongodb/pull/65/commits/163a951857899ac70ee2944a892eb1b6eda2a484